freqtrade/optimize/hyperopt.py

```diff
@@ -686,16 +715,23 @@ def start(self) -> None:
                         # evaluations can take different time. Here they are aligned in the
                         # order they will be shown to the user.
                         val['is_best'] = is_best
-
-                        self.print_results(val)
+                        output = self.get_results(val)
+                        if output:
+                            self.progress_bar.write(output)
+                        self.progress_bar.ncols = 108
```

Comment:
```suggestion
```
-- not needed

Reply:
In my testing it did not keep the ncols during the threaded part. 
But will test again

Reply:
printing is done in one main process, not in the workers

Reply:
Hmm it didn't work in my initial tests but i guess with all the changes i made over time it's working now.. just tested it